### ISSUE TITLE
RuntimeError: generator raised StopIteration

### ISSUE THREAD

**Issue by fluxrider** [golightlyb/bakefont3#1]
https://github.com/golightlyb/bakefont3/blob/b5e05f5f96dc37136cf1cf6053c081a7b30f9ea8/bakefont3/encode.py#L279

Hello, first time user of bakefont3 here. I'm running into a StopIteration error when running the example. It looks like something changed in more recent versions of python in the last 3 years, that broke your code. There seems to be a very simple fix, which works for me anyway, that I got from this thread: https://stackoverflow.com/questions/37707187/python-pep479-change-stopiteration-handling-inside-generators

In a nutshell, replace 'raise StopIteration' with a simple 'return'.

I'm using Python 3.9.3 on Arch Linux Arm.

Cheers,

**Comment by golightlyb** [comment]
Thank you very much for reporting this with a fix! I will update the repo shortly.
